test-clients/src/main/java/com/hedera/services/bdd/spec/HapiApiSpec.java

```diff
@@ -327,7 +327,7 @@ private void startFinalizingOps() {
 										try {
 											op.finishFor(this);
 										} catch (Throwable t) {
-											log.warn(logPrefix() + op + " failed!", t);
+											log.warn(logPrefix() + op + " failed!");
```

Comment:
When we don't log an exception/throwable, rename it to `ignore`.

Reply:
We still pack the exception message to let the finalizer to handle it.